supabase/authentication/email_templates/confirm_signup.html

```diff
@@ -0,0 +1,50 @@
+          </td>
+        </tr>
+        <tr>
+          <td style="padding:36px 30px 42px 30px;">
+            <table style="width:100%;border-collapse:collapse;border:0;border-spacing:0;" role="presentation">
+              <tbody><tr>
+                <td style="padding:0 0 36px 0;color:#153643;">
+                  <h1 style="font-size:24px;margin:0 0 20px 0;font-family:Arial,sans-serif;">Welcome to Motis Ride</h1>
+                  <p style="margin:0 0 12px 0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;">To finish setting up your account, please click on the link below. Scroll down to see what Motis Ride can do for you!</p>
+                  <p style="margin:0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;"><a style="color:#ee4c50;text-decoration:underline;" href="{{ .ConfirmationURL }}">Take me back to the app</a></p>
+                </td>
+              </tr>
+              <tr>
+                <td style="padding:0;">
+                  <table style="width:100%;border-collapse:collapse;border:0;border-spacing:0;" role="presentation">
+                    <tbody><tr>
+                      <td style="width:260px;padding:0;vertical-align:top;color:#153643;">
+                        <p style="margin:0 0 25px 0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;"><img style="height:auto;display:block;" width="260" alt="" src="https://assets.codepen.io/210284/left.gif"></p>
+                        <p style="margin:0 0 12px 0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;">Offering rides has never been easier! Just enter your route details and we will automatically find the best combination of riders for you! </p>
+                      </td>
+                      <td style="width:20px;padding:0;font-size:0;line-height:0;">&nbsp;</td>
+                      <td style="width:260px;padding:0;vertical-align:top;color:#153643;">
+                        <p style="margin:0 0 25px 0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;"><img style="height:auto;display:block;" width="260" alt="" src="https://assets.codepen.io/210284/right.gif"></p>
+                        <p style="margin:0 0 12px 0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;">Finding a ride has never been easier! Just enter where you want to go and we will automatically find the best driver for you!</p>
+                      </td>
+                    </tr>
+                  </tbody></table>
+                </td>
+              </tr>
+            </tbody></table>
+          </td>
+        </tr>
+        <tr>
+          <td style="padding:30px;background:#ee4c50;text-align: center;vertical-align: middle;">
+            <p style="margin:0;font-size:16px;line-height:24px;font-family:Arial,sans-serif;color:#ffffff;">© Motis Mitfahr-App Team 2023</p>
+          </td>
+        </tr>
+      </tbody></table>
+    </td>
+  </tr>
+</tbody></table>
```

Comment:
Can you put a newline at the end of the file?

Reply:
There was one, but supabase always removes it for some reason...

Reply:
So to be as consistent as possible, we should probably leave it this weird way